pep-0001.txt

```diff
@@ -455,15 +460,26 @@ Each PEP should have the following parts:
    The final implementation must include test code and documentation
    appropriate for either the Python language reference or the
    standard library reference.
-
-9. How to Teach This -- For a PEP that adds new functionality or changes
-   language behavior, it is helpful to include a section on how to
-   teach users, new and experienced, how to apply the PEP to their
-   work.
-
-   This section may include key points and recommended documentation
-   changes that would help users adopt a new feature or migrate their
-   code to use a language change.
+   
+9. Rejected Ideas -- Throughout the discussion of a PEP, various ideas
```

Comment:
(Aside: this is in a sense part of the rationale -- it contains the rationale for rejecting various alternatives that were brought up. I do agree it is better done as a separate section.)